.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 88
```

Comment:
Pylons Project projects have a [hard rule of 79-column line length](https://pylonsproject.org/community-coding-style-standards.html) under "Coding Style".

See also Pyramid's .flake8 for possibly reusable bits.
```
[flake8]
ignore =
    # E203: whitespace before ':' (black fails to be PEP8 compliant)
    E203
    # E731: do not assign a lambda expression, use a def
    E731
    # W503: line break before binary operator (flake8 is not PEP8 compliant)
    W503
    # W504: line break after binary operator (flake8 is not PEP8 compliant)
    W504
exclude =
    src/pyramid/compat.py
    tests/fixtures
    tests/pkgs
    tests/test_config/pkgs
    tests/test_config/path
show-source = True
```